db/migrate/20160605191612_create_lists.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLists < ActiveRecord::Migration
+  def change
+    create_table :lists do |t|
+      t.string :name, null: false, unique: true
```

Comment:
can we take out the unique true. It prevents any user from having the same title like Food from home.